src/main/java/com/efficio/fieldbook/web/common/service/impl/CrossingServiceImpl.java

```diff
@@ -605,7 +605,7 @@ Name createName(final Germplasm germplasm, final ImportedCross cross, final Inte
 		// exist as crosses are created in crossing manager and persisted.
 		if (cross.getGid() != null) {
 			final List<Progenitor> existingProgenitors = this.pedigreeDataManager.getProgenitorsByGID(Integer.valueOf(cross.getGid()));
-			if (existingProgenitors != null) {
```

Comment:
Use `CollectionUtils`.isEmpty to prevent NPE

Reply:
In this case how it can be null if I am returnin criteria.list? I am preventing NPE from the DAO directly with my change in ProgenitorDAO